example-monorepos/with-tailwindcss/apps/next/pages/_app.tsx

```diff
@@ -4,6 +4,9 @@ import React from 'react'
 import type { SolitoAppProps } from 'solito'
 import 'raf/polyfill'
 
+import 'app/rnw-overrides'
+import 'app/tailwindcss-react-native.d'
```

Comment:
type declaration shouldn’t go in code i believe

Reply:
I'm having difficulty compiling the example for production without them. It seems to work fine in development but once I do I production build it fails. I pushed this up to test on another computer

Reply:
you might need to put them in the root and then do a file that uses references like `next.d.ts`

Reply:
I can replicate with with the `blank` example on `master`. If you add `accessibilityLevel?: number` to `TextProps` and add that prop to `Text` the project fails to build.

Reply:
I'm running out of time to look into this today, so I'll  have to continue looking at this over the weekend

```diff
diff --git a/example-monorepos/blank/packages/app/features/home/screen.tsx b/example-monorepos/blank/p
ackages/app/features/home/screen.tsx
index 6620d28..dd367ca 100644
--- a/example-monorepos/blank/packages/app/features/home/screen.tsx
+++ b/example-monorepos/blank/packages/app/features/home/screen.tsx
@@ -59,6 +59,7 @@ export function HomeScreen() {
           }}
         >
           <Text
+            fakeProp="asdf"
             selectable={false}
             sx={{ fontSize: 16, color: 'black', fontWeight: 'bold' }}
           >
diff --git a/example-monorepos/blank/packages/app/rnw-overrides.tsx b/example-monorepos/blank/packages
/app/rnw-overrides.tsx
index 6d2767c..c3a8120 100644
--- a/example-monorepos/blank/packages/app/rnw-overrides.tsx
+++ b/example-monorepos/blank/packages/app/rnw-overrides.tsx
@@ -11,6 +11,7 @@ declare module 'react-native' {
     transitionDuration?: string
   }
   interface TextProps {
+    fakeProp?: string
     accessibilityComponentType?: never
     accessibilityTraits?: never
     href?: string
```

That's the diff I'm doing on the master branch. Then if you `cd example-monorepos/blank/apps/next && yarn build` the build will fail.

Reply:
Good call on `next-env.d.ts`. I mirrored that pattern and everything seems to work without any issues
 https://github.com/nandorojo/solito/pull/72/commits/0afee93c81e9e5f61f6ba849342e3fb5bab8c016